fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -419,6 +419,7 @@ ACTOR Future<Void> commitBatcher(ProxyCommitData* commitData,
 					}
 
 					++commitData->stats.txnCommitIn;
+					commitData->stats.uniqueClients.insert(cpi.address());
```

Comment:
Isn't the the address associated with the CommitProxyInterface the server address, not the client address?

I don't know if the client address comes with the request, but you can probably get it from req.reply.getEndpoint().getPrimaryAddress(), which is what I see  other code in this function doing.

Reply:
This is a very good catch :)